test/test.js

```diff
@@ -113,4014 +93,14 @@ afterAll(async({server, httpsServer}) => {
-      });
-      expect(result).toBe(15);
-    });
-    it('should work on frames before navigation', async({page, server}) => {
-      await page.goto(server.PREFIX + '/frames/nested-frames.html');
-      await page.exposeFunction('compute', function(a, b) {
-        return Promise.resolve(a * b);
-      });
-
-      const frame = page.frames()[1];
-      const result = await frame.evaluate(async function() {
-        return await compute(3, 5);
-      });
-      expect(result).toBe(15);
-    });
-  });
-
-  describe('Page.setRequestInterception', function() {
-    it('should intercept', async({page, server}) => {
-      await page.setRequestInterception(true);
-      page.on('request', request => {
-        expect(request.url()).toContain('empty.html');
-        expect(request.headers()['user-agent']).toBeTruthy();
-        expect(request.method()).toBe('GET');
-        expect(request.postData()).toBe(undefined);
-        expect(request.resourceType()).toBe('document');
-        expect(request.frame() === page.mainFrame()).toBe(true);
-        expect(request.frame().url()).toBe('about:blank');
-        request.continue();
-      });
-      const response = await page.goto(server.EMPTY_PAGE);
-      expect(response.ok()).toBe(true);
-    });
-    it('should stop intercepting', async({page, server}) => {
-      await page.setRequestInterception(true);
-      page.once('request', request => request.continue());
-      await page.goto(server.EMPTY_PAGE);
-      await page.setRequestInterception(false);
-      await page.goto(server.EMPTY_PAGE);
-    });
-    it('should show custom HTTP headers', async({page, server}) => {
-      await page.setExtraHTTPHeaders({
-        foo: 'bar'
-      });
-      await page.setRequestInterception(true);
-      page.on('request', request => {
-        expect(request.headers()['foo']).toBe('bar');
-        request.continue();
-      });
-      const response = await page.goto(server.EMPTY_PAGE);
-      expect(response.ok()).toBe(true);
-    });
-    it('should works with customizing referer headers', async({page, server}) => {
-      await page.setExtraHTTPHeaders({ 'referer': server.EMPTY_PAGE });
-      await page.setRequestInterception(true);
-      page.on('request', request => {
-        expect(request.headers()['referer']).toBe(server.EMPTY_PAGE);
-        request.continue();
-      });
-      const response = await page.goto(server.EMPTY_PAGE);
-      expect(response.ok()).toBe(true);
-    });
-    it('should be abortable', async({page, server}) => {
-      await page.setRequestInterception(true);
-      page.on('request', request => {
-        if (request.url().endsWith('.css'))
-          request.abort();
-        else
-          request.continue();
-      });
-      let failedRequests = 0;
-      page.on('requestfailed', event => ++failedRequests);
-      const response = await page.goto(server.PREFIX + '/one-style.html');
-      expect(response.ok()).toBe(true);
-      expect
```

Comment:
let's require files explicitly; will work just fine for now, will save us a dependency and will be nicely grepp'able.

Reply:
(this comment was about using the `glob` dependency and listing all the `*.spec.js` files)

Reply:
Fixed. Requiring tests explicitly now. The reason I used glob is so we won't have to explicitly add each new test file to some list of test files...